source/2020-01-27-the-ember-times-issue-132.md

```diff
@@ -24,12 +24,14 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Get together at EmberConf meetups 🍦🎲🗻](https://emberconf.com/#/schedule/meetups)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Besides learning a lot of the talks at EmberConf it is also an amazing place to meet people and get to know them. One of the easiest places to do that is at the meet-ups that can be organised by **you**. 
+
+Last year there were a lot of [different meet-ups](https://2019.emberconf.com/meetups.html), from delicious food and drink gatherings to a nerve-racking chess tournament and bouldering. 
+
+If you have a great idea for a meet-up, you can [sign up here](https://emberconf.com/#/schedule/meetups). Keep an eye on this page as well if you want to know about all the meet-ups and when they will happen.
```

Comment:
```suggestion
If you have a great idea for a meetup, you can [sign up here](https://emberconf.com/#/schedule/meetups). Keep an eye on this page as well if you want to know about all the meetups and when they will happen.
```

i think meetup is fine either way, but changed it to be consistent! 

other suggestions are for clarity, up to you!